src/locale/locales/hu/messages.po

```diff
@@ -0,0 +1,7805 @@
+
+#: src/screens/Profile/Header/ProfileHeaderLabeler.tsx:228
+msgid "Subscribe to Labeler"
+msgstr "Feliratkozás a feljegyzőre"
+
+#: src/screens/Profile/Header/ProfileHeaderLabeler.tsx:194
+msgid "Subscribe to this labeler"
+msgstr "Feliratkozás a feljegyzőre"
+
+#: src/view/screens/ProfileList.tsx:699
+msgid "Subscribe to this list"
+msgstr "Feliratkozás a listára"
+
+#: src/components/dialogs/VerifyEmailDialog.tsx:81
+msgid "Success!"
+msgstr "Siker!"
+
+#: src/view/screens/Search/Explore.tsx:332
+msgid "Suggested accounts"
+msgstr "Javasolt fiókok"
+
+#: src/components/FeedInterstitials.tsx:318
+msgid "Suggested for you"
+msgstr "Számodra javasolt"
+
+#: src/view/com/composer/labels/LabelsBtn.tsx:166
+#: src/view/com/composer/labels/LabelsBtn.tsx:169
+msgid "Suggestive"
+msgstr "Felnőtt tartalom"
+
+#: src/Navigation.tsx:260 src/view/screens/Support.tsx:31
+#: src/view/screens/Support.tsx:34
+msgid "Support"
+msgstr "Támogatás"
+
+#: src/components/dialogs/SwitchAccount.tsx:46
+#: src/components/dialogs/SwitchAccount.tsx:49
+msgid "Switch Account"
+msgstr "Fiókváltás"
+
+#: src/view/screens/Settings/index.tsx:131
+msgid "Switch to {0}"
+msgstr "Váltás rá: {0}"
+
+#: src/view/screens/Settings/index.tsx:132
+msgid "Switches the account you are logged in to"
+msgstr "A jelenleg bejelentkezett fiók átváltása egy másikra"
+
+#: src/components/dialogs/nuxs/NeueTypography.tsx:78
+#: src/screens/Settings/AppearanceSettings.tsx:101
+#: src/screens/Settings/AppearanceSettings.tsx:148
+msgid "System"
+msgstr "Rendszer"
+
+#: src/view/screens/Settings/index.tsx:818
+msgid "System log"
+msgstr "Rendszernapló"
+
+#: src/components/TagMenu/index.tsx:87
+msgid "Tag menu: {displayTag}"
+msgstr "Címkemenü: {displayTag}"
+
+#: src/components/dialogs/MutedWords.tsx:282
+msgid "Tags only"
+msgstr "Csak címkék"
+
+#: src/components/ProgressGuide/Toast.tsx:150
+msgid "Tap to dismiss"
+msgstr "Koppints ide a bezáráshoz"
+
+#: src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerNative.tsx:136
+msgid "Tap to enter full screen"
+msgstr "Koppints ide a teljes képernyős módba lépéshez"
+
+#: src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerNative.tsx:142
+msgid "Tap to play or pause"
+msgstr "Koppints ide a lejátszáshoz/szüneteltetéshez"
+
+#: src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerNative.tsx:159
+msgid "Tap to toggle sound"
+msgstr "Koppints ide a hang némításához és felhangosításához"
+
+#: src/view/com/util/images/AutoSizedImage.tsx:219
+#: src/view/com/util/images/AutoSizedImage.tsx:239
+msgid "Tap to view full image"
+msgstr "Koppints ide a teljes kép megtekintéséhez"
+
+#: src/state/shell/progress-guide.tsx:166
+msgid "Task complete - 10 likes!"
+msgstr "Feladat teljesítve – 10 kedvelés!"
+
+#: src/components/ProgressGuide/List.tsx:49
+msgid "Teach our algorithm what you like"
+msgstr ""
+"Tanítsd meg az algoritmusnak,\n"
+"hogy mit szeretnél látni!"
```

Comment:
```suggestion
msgstr "Tanítsd meg az algoritmusnak, hogy mit szeretnél látni!"
```
I think there was a bit of a glitch here

Reply:
This was intentional. If I don't specify a line break there, the sidebar doesn't do it by itself and expands to the right, adding a horizontal scroll bar.

Reply:
Oh I see, I didn’t realise it was possible to add a line break like that

Reply:
We've tested it and it works. 👍

Reply:
That’s what counts 👌